pytest-pixl/src/pytest_pixl/dicom.py

```diff
@@ -111,6 +120,9 @@ def generate_dicom_dataset(tag_values: dict = TAGS_DICT, **kwargs: Any) -> Datas
             msg = f"Tag {key} is not a valid DICOM tag"
             raise ValueError(msg)
 
+    if "Modality" in kwargs and "SOPClassUID" not in kwargs:
+        ds.SOPClassUID = MODALITY_TO_CLASS_UID[kwargs["Modality"]]
```

Comment:
interesting that this isn't covered by a test. is coverage just playing up?

Reply:
hmm, yeah I think it must be covered as without it the tests fail